data_struct.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+#include <string.h>
+
+struct Car_type
+
+{
+    char model[30];
+    char branch[30];
+    int years;
+};
+
+struct Car_type2
+{
+    char model[30];
+    char branch[30];
+    int years;
+};
```

Comment:
Agreed  -- it would be better if you created a type and applied it...
```suggestion
typedef struct
{
    char model[30];
    char branch[30];
    int years;
}CarType_t;
```